src/service/viewer-impl.js

```diff
@@ -74,6 +74,12 @@ export const VisibilityState = {
 };
 
 
+/** @export {!Array<string>} */
+export const TRUSTED_VIEWER_HOSTS = [
```

Comment:
Add docs what the rough process is to add a viewer here (file an issue and we discuss).